README.md

```diff
@@ -4,45 +4,65 @@ The dataplane-operator automates the deployment of an OpenStack dataplane.
 ---
 **NOTE**
 
-The OpenStackDataPlane CRD/controller will eventually be moved to
+The OpenStackDataPlane CRD/controller may eventually be moved to
 [openstack-operator](https://github.com/openstack-k8s-operator/openstack-operator).
 The move is so that openstack-operator, as the umbrella operator, remains the
 single managing operator and UX for a complete OpenStack deployment
 (control plane and data plane).
 
 ---
 
+## DataPlane, Roles, and Nodes
+
 The dataplane-operator exposes the concepts of data plane role and nodes. These
 are represented as CRD's within the operator:
 
+* [OpenStackDataPlane](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/crd/bases/dataplane.openstack.org_openstackdataplane.yaml)
 * [OpenStackDataPlaneRole](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/crd/bases/dataplane.openstack.org_openstackdataplaneroles.yaml)
 * [OpenStackDataPlaneNode](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/crd/bases/dataplane.openstack.org_openstackdataplanenodes.yaml)
 
-The OpenStackDataPlaneRole CRD provides for a logical grouping of nodes of a
-similar type within a role.  Similarities within a role are defined by the
-user, and could be of a small scope (ansible port), or a large scope (same
-network config, nova config, provisioning config, etc).
-
-A role also provides for an inheritance model of node attributes. Attributes on
-the role will automatically be inherited by the nodes on that role. Nodes can
-also set their own attributes, which will override the inherited values from
-the role.
+The OpenStackDataPlane CRD provides for a logical grouping of roles, that
+altogether make up an OpenStack dataplane.
 
-The OpenStackDataPlaneRole controller creates and updates a node CR for each
-node in that role. It also provides synchronization points as its nodes are
-deployed.
+Roles are grouped into the dataplane with the `DataPlane` field set on the
+role. Roles are also labeled with the dataplane they belong to:
 
-The OpenStackDataPlaneNode CRD and controller encapsulates all the provisioning
-and deployment logic for a single node. The OpenStackDataPlaneNode controller
-operates on a single node CR, and manages that node in isolation, performing
-all the necessary baremetal provisioning and software deployment/configuration
-on that node.
+`openstackdataplane=<dataplane-name>`
 
-The OpenStackDataPlane CRD and controller orchestrates deploying roles, and
-provides synchronization points between role deployment.
+The dataplane does not maintain the list of roles in that dataplane, to ensure
+that the source source of dataplane and role grouping is stored only in the
+`DataPlane` field on the role.
```

Comment:
Perhaps re-write 32,33,34 as:

The dataplane does not maintain a list of roles. The source of dataplane and role grouping is stored in the `DataPlane` field of the role.

Reply:
Done.